recipes-bsp/firmware-imx/firmware-ele-imx_0.1.0.bb

```diff
@@ -0,0 +1,32 @@
+# Copyright 2021-2023 NXP
+SUMMARY = "NXP i.MX ELE firmware"
+DESCRIPTION = "EdgeLock Enclave firmware for i.MX series SoCs"
+SECTION = "base"
+LICENSE = "Proprietary"
+LIC_FILES_CHKSUM = "file://COPYING;md5=2827219e81f28aba7c6a569f7c437fa7"
+
+inherit fsl-eula-unpack use-imx-security-controller-firmware deploy
+
+RREPLACES:${PN} = "firmware-sentinel"
```

Comment:
The RREPLACES and RPROVIDES should be bellow the FILES.

Reply:
#resolved pushed